controllers/automation/packagedeployment_controller.go

```diff
@@ -230,11 +236,16 @@ metadata:
 	// If a binding resource exists in the package, and one exists
 	// that matches the name of the associated cluster, overwrite the
 	// resource in the package with the cluster resource
+	conditions := convertConditions(newPR.Status.Conditions)
 
 	for id, pkgResource := range bindingResources {
 		clusterResource, err := r.findClusterObject(ctx, c, id)
+		group, _ := resid.ParseGroupVersion(id.APIVersion)
+		conditionType := fmt.Sprintf("%s.%s.%s.%s.Injected", group, id.Kind, c.Name, c.Namespace)
 		if err != nil {
-			r.l.Info(fmt.Sprintf("error finding cluster resource: %s", err.Error()), "cluster", c)
+			r.l.Info(fmt.Sprintf("error looking for cluster resource: %s", err.Error()), "cluster", c)
```

Comment:
In a subsequent PR (shouldn't block this, but maybe create an issue), we should consider emitting a K8s event for injections (succeeded or not).